rust/src/mime/smtp_log.rs

```diff
@@ -197,21 +197,22 @@ pub unsafe extern "C" fn SCMimeSmtpLogFieldString(
 }
 
 fn log_data_header(
-    js: &mut JsonBuilder, ctx: &MimeStateSMTP, hname: &str,
+    js: &mut JsonBuilder, ctx: &MimeStateSMTP, hname: &str, field: &str,
 ) -> Result<(), JsonError> {
     for h in &ctx.headers[..ctx.main_headers_nb] {
         if mime::slice_equals_lowercase(&h.name, hname.as_bytes()) {
-            js.set_string(hname, &String::from_utf8_lossy(&h.value))?;
+            js.set_string(field, &String::from_utf8_lossy(&h.value))?;
             break;
         }
     }
     return Ok(());
 }
 
 fn log_data(js: &mut JsonBuilder, ctx: &MimeStateSMTP) -> Result<(), JsonError> {
-    log_data_header(js, ctx, "from")?;
-    log_data_header(js, ctx, "date")?;
-    log_data_header(js, ctx, "subject")?;
+    log_data_header(js, ctx, "from", "from")?;
+    log_data_header(js, ctx, "date", "date")?;
+    log_data_header(js, ctx, "subject", "subject")?;
+    log_data_header(js, ctx, "message-id", "message_id")?;
```

Comment:
Looking at https://github.com/OISF/suricata/blob/master/src/output-json-email-common.c#L69 this shouldn't be logged by default, but only in the extended mode or in custom mode.

Reply:
Ok, should I also remove subject? ``date`` is ``LOG_EMAIL_DEFAULT`` should I keep it or remove it as well?

Reply:
I think so ya